src/blocks/page.css

```diff
@@ -0,0 +1,33 @@
+@font-face {
```

Comment:
`index.css` file name better suits

Reply:
Thank you, already moved font-faces to index.css